### PR TITLE
Change default priority on autoconfiguration Doctrine DataProvider extensions

### PR DESCRIPTION
**Pull request by Mathieudewet**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #1815
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm.xml [hunk 1]
```diff
@@ -153,15 +153,15 @@
             <argument>%api_platform.collection.pagination.client_partial%</argument>
             <argument>%api_platform.collection.pagination.partial_parameter_name%</argument>
 
-            <tag name="api_platform.doctrine.orm.query_extension.collection" priority="8" />
+            <tag name="api_platform.doctrine.orm.query_extension.collection" priority="-64  " />
```
Comment (Simperfit) [inline src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm.xml]: `-64  ` -> `-64`

**Comment by Simperfit** [comment]
Thanks @Mathieudewet.

**Comment by soyuka** [comment]
can you reopen this @Simperfit ?

**Comment by Simperfit** [comment]
I restored the branch, But the PR cannot be re-open a new one should be created.

**Pull request merged**